docs/sphinx/using/cudaq/platform.rst

```diff
@@ -104,32 +115,90 @@ should be used.
-    :language: cpp
-    :start-after: [Begin Documentation]
-    :end-before: [End Documentation]
+    .. literalinclude:: ../../snippets/cpp/using/cudaq/platform/observe_mqpu_mpi.cpp
+        :language: cpp
+        :start-after: [Begin Documentation]
+        :end-before: [End Documentation]
 
-.. code-block:: console
+    .. code-block:: console
 
-    nvq++ file.cpp -target nvidia-mqpu
-    mpirun -np <N> a.out
+        nvq++ file.cpp -target nvidia-mqpu
+        mpirun -np <N> a.out
 
 
-Python
-^^^^^^
+.. tab:: Python
 
-.. literalinclude:: ../../snippets/python/using/cudaq/platform/observe_mqpu_mpi.py
-    :language: python
-    :start-after: [Begin Documentation]
+    .. literalinclude:: ../../snippets/python/using/cudaq/platform/observe_mqpu_mpi.py
+        :language: python
+        :start-after: [Begin Documentation]
 
-.. code-block:: console
+    .. code-block:: console
 
-    mpirun -np <N> python3 file.py
+        mpirun -np <N> python3 file.py
 
 In the above example, the parallel distribution mode was set to :code:`mpi` using :code:`cudaq::parallel::mpi` in C++ or :code:`cudaq.parallel.mpi` in Python.
 CUDA Quantum provides MPI utility functions to initialize, finalize, or query (rank, size, etc.) the MPI runtime. 
 Last but not least, the compiled executable (C++) or Python script needs to be launched with an appropriate MPI command, 
 e.g., :code:`mpirun`, :code:`mpiexec`, :code:`srun`, etc.
+
+Remote REST Server Platform
++++++++++++++++++++++++++++
+
+The remote simulator target (:code:`remote-sim`) encapsulates simulated QPUs
+as independent REST server instances, to which the CUDA Quantum runtime communicating 
+to via HTTP requests (REST API).
+
+CUDA Quantum provides the REST server implementation as a standalone application (:code:`cudaq_rest_server`)
+hosting all the simulator backends available in the installation, including those that require MPI for multi-GPU computation.
+To start the server, serving at a specific TCP/IP port, one can do the following.
+
+.. code-block:: console
+    cudaq_rest_server --port <port number>
+
+User code can then target this platform by specifying its target name (:code:`remote-sim`).
+
+.. tab:: C++
+
+    .. code-block:: console
+
+        nvq++ file.cpp --target remote-sim --remote-sim-url <url1[,url2,...]> --remote-sim-backend <sim1[,sim2,...]>
+
+
+.. tab:: Python
+
+     .. code:: python 
+
+        cudaq.set_target("remote-sim", url="url1[,url2,...]", backend="sim1[,sim2,...]")
+    
+
+When using this target, the user needs to provides a list of URLs where (:code:`cudaq_rest_server`) is serving.
+The number of QPUs (:code:`num_qpus()`) is equal to the number of URLs provided. 
+
+Each QPU instance can be assigned a different backend simulator via the :code:`--remote-sim-backend` (`nvq++`) or :code:`backend` (Python)
+option. Otherwise, if a single backend is specified, all the QPUs are assumed to be using the same simulator.
+
+Auto-launch REST Server
```

Comment:
If the documentation is in this order, there is a good chance people will not read about the auto-launch feature before attempting (and possibly struggling with) making the manual launch work. For documentation, always start with the simplest possible scenario first (in this case, using auto-launch), and then elaborate on more advanced uses (when would/should a user use manual launch?).

Reply:
I've updated the docs to follow the suggested flow: auto-launch first then manual launch.